docs/magento2/commerce.mdx

```diff
@@ -185,3 +212,180 @@ filters on ` option_id` instead of `value_id`.
+  value: "0"
+}
+```
+
+When the user selects the `Yes` option, the value `1` is displayed in the return
+details page instead of the option label `Yes`.
+
+<BrowserWindow noPadding>
+
+<video width="100%" height="400" controls className="video video-contain">
+  <source
+    src={require("./assets/rma-yes-no-limitation.mp4").default}
+    type="video/mp4"
+  />
+  Your browser does not support the video tag.
+</video>
+
+</BrowserWindow>
+
+This is due to the way the Magento Commerce module returns the submitted values,
+we do not currently have any way of detecting that the field is a boolean field
+to eventually map the values to their labels.
+
+#### Possible solutions
+
+##### 1. Use a dropdown field (recommended)
+
+Use the dropdown field with the options `Yes` and `No`. The dropdown field
+correctly use the option values instead of the option ids.
+
+##### 2. Override the `ReturnedItemAttributeTable`
+
+You can override the `ReturnedItemAttributeTable` component to map the values
+based on a static label, for example
+
+```jsx title="src/web/theme/modules/User/Order/OrderReturns/ReturnedItemAttributeTable.js"
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import Wysiwyg from "theme/modules/WysiwygV2";
+
+/**
+ * @typedef Attribute
+ * @property {string} label
+ * @property {TODO_TypeDef} value
+ */
+
+/**
+ * @type {React.FC<{ attributes:Attribute[] }>} props
+ */
+const ReturnedItemAttributeTable = (props) => {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>
+            <FormattedMessage
+              id="modules.User.Order.OrderReturns.ReturnedItemAttributeTable.question"
+              defaultMessage="Question"
+            />
+          </th>
+          <th>
+            <FormattedMessage
+              id="modules.User.Order.OrderReturns.ReturnedItemAttributeTable.answer"
+              defaultMessage="Answer"
+            />
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.attributes.map((attribute, i) => (
+          <tr key={attribute.label + `-${i}`}>
+            <td>{attribute.label}</td>
+            <td>
+              // remove-next-line
+              <Wysiwyg content={attribute.value} />
+              // add-start
+              {attribute.label === "Yes/No" ? (
+                attribute.value === "1" ? (
+                  "Yes"
+                ) : (
+                  "No"
+                )
+              ) : (
+                <Wysiwyg content={attribute.value} />
+              )}
+              // add-end
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  );
+};
+
+export default ReturnedItemAttributeTable;
+```
+
+```mdx-code-block
+  </div>
+</details>
+```
+
+<!--------------------DETAILS-------------------->
+
+```mdx-code-block
+<details>
+  <summary className="flex justify-between"><span>The <code>File and Image</code> fields are not supported</span><SinceVersion inline tag="2.22.1" /></summary>
```

Comment:
same here on the version.

I would even argue that it's not a known issue but rather the explanation of why File and Image are not supported so maybe you could move the sentence _Magento is not yet ready to handle such files in its current state._ to the table on Field type support.